README.md

```diff
@@ -34,29 +34,6 @@ make test
 See [`CONTRIBUTING.md`](CONTRIBUTING.md) for details on how to contribute to
 this project.
```

Comment:
this project, including how to build and test your changes as well as how to properly format your code.